### DISCUSSION TITLE
Help to artist

### DISCUSSION THREAD

**Discussion by igorraand** [OpenRarity/open-rarity#86] [Q&A] [answered]
Hi guys, I'm just an artist. I want to connect Openrarity to my collection on Opensea. but I absolutely don't understand how to do it all I found is some code. maybe I look stupid, but I do not know how to connect it, I will be glad of your help.
I want to add rarity to my NFT

**Comment by impreso** [comment, marked as answer]
@igorraand Please follow this guide here https://support.opensea.io/hc/en-us/articles/9517355517843-What-is-OpenRarity- . Detailed instructions how to opt-in to OpenRarity.

**Reply by igorraand** [reply]
I read this, i  put screenshot what i have,  and wanna ask how to  exactly connect
this  OpenRarity to make this ERC-721 work. 
<img width="851" alt="Screenshot 2022-10-21 at 12 02 49" src="https://user-images.githubusercontent.com/60923467/197108959-a96961e3-e936-4049-8e92-fd695b7c9315.png">

**Comment by igorraand** [comment]
i follow this guide but need connect OpenRarity  Api. connect this  "ERC721".
i donk know how to connet this. and cant find guide in internet. Ofc i find
Github page, but i see many lines of code and nothing know how to use it.
[image: Screenshot 2022-10-21 at 11.59.52.png]

пт, 21 окт. 2022 г. в 04:02, Dan Meshkov ***@***.***>:

**Comment by impreso** [comment]
Can you send us your opensea collection link?

**Reply by igorraand** [reply]
https://opensea.io/collection/nakeddaria

**Reply by igorraand** [reply]
> Can you send us your opensea collection link?

i send can you check please

**Reply by impreso** [reply]
Yeah sure - you have a numeric attribute in your collection ( examples are Body, Cool, Nice). We will support numeric attributes in the next OpenRarity release.

**Reply by igorraand** [reply]
what the attribute support right now? where i can see?
